site/_posts/2018-07-13-parameter-server-in-fifteen-lines.markdown

```diff
@@ -127,26 +125,32 @@ array([64., 64., 64., 64., 64., 64., 64., 64., 64., 64.])
 array([78., 78., 78., 78., 78., 78., 78., 78., 78., 78.])
 ```
 
+Part of the value that Ray adds here is that *Ray makes it as easy to start up a
+remote service or actor as it is to define a Python class*. Handles to the actor
+can be passed around to other actors and tasks to allow arbitrary messaging and
+communication patterns. Current alternatives are much more involved. For
+example, [consider how this would be done with GRPC][14].
```

Comment:
Yet another nit, but I think using "this" instead of what it stands for (which is "runtime grpc server handle passing") makes the statement lose a bit of force, but it might be a bit too complicated to express this